frontend/src/selectors/staticContentSelectors.js

```diff
@@ -103,27 +109,49 @@ export const getDropdownIncidentFollowupActionOptions = createSelector(
 );
 
 export const getDropdownHSRCMComplianceCodes = createSelector(
-  [getComplianceCodes],
+  [getCurrentComplianceCodes],
   (codes) =>
     codes
       .filter(({ article_act_code }) => article_act_code === "HSRCM")
       .map((code) => {
-        const composedLabel = `${code.section}.${code.sub_section}.${code.paragraph} - ${
-          code.description
-        }`;
+        const composedLabel = code.sub_paragraph
+          ? `${code.section}.${code.sub_section}.${code.paragraph}.${code.sub_paragraph} - ${
+              code.description
+            }`
+          : `${code.section}.${code.sub_section}.${code.paragraph} - ${code.description}`;
         return { value: code.compliance_article_id, label: composedLabel };
       })
 );
 
+export const getMultiSelectComplianceCodes = createSelector(
+  [getCurrentComplianceCodes],
+  (codes) =>
+    codes
+      .filter(({ article_act_code }) => article_act_code === "HSRCM")
+      .map((code) => {
+        const composedValue = code.sub_paragraph
+          ? `${code.section}.${code.sub_section}.${code.paragraph}.${code.sub_paragraph}`
+          : `${code.section}.${code.sub_section}.${code.paragraph}`;
+        const composedLabel = code.sub_paragraph
+          ? `${code.section}.${code.sub_section}.${code.paragraph}.${code.sub_paragraph} - ${
+              code.description
+            }`
+          : `${code.section}.${code.sub_section}.${code.paragraph} - ${code.description}`;
+        return { value: composedValue, label: composedLabel };
+      })
+);
+
 export const getHSRCMComplianceCodesHash = createSelector(
-  [getComplianceCodes],
+  [getCurrentComplianceCodes],
   (codes) =>
     codes
       .filter(({ article_act_code }) => article_act_code === "HSRCM")
       .reduce((map, code) => {
-        const composedValue = `${code.section}.${code.sub_section}.${code.paragraph} - ${
-          code.description
-        }`;
+        const composedValue = code.sub_paragraph
+          ? `${code.section}.${code.sub_section}.${code.paragraph}.${code.sub_paragraph} - ${
```

Comment:
Also here. And it may be worth it to just pull the `composed*` function out of the exported functions so that they can all share it (assuming they all use the same logic)